packages/design-system/src/styles/components/_Table.scss

```diff
@@ -55,37 +61,60 @@
     word-wrap: normal;
   }
 
-  th,
-  td {
+  tr,
+  caption {
+    display: block;
+  }
+
+  td,
+  th {
     border: 0;
+    display: grid;
+    grid-template-columns: 42% auto;
```

Comment:
Why is this 42 instead of 40?

Reply:
I think setting to 42% starts the second column to the right a bit to mimic the left cell padding. 
![Screen Shot 2020-08-31 at 4 21 08 PM](https://user-images.githubusercontent.com/1449852/91764845-fc1b4c00-eba5-11ea-9c3c-7af70e407e63.png) 

If it is set to 40% the text is too close to the grey background and is not spaced evenly.

Reply:
hm, is there a better way of styling that cell padding? I could see this being confusing or hard to customize for folks

Reply:
Yes, updated to `grid-template-columns: 120px 1fr`